.github/workflows/lint.yaml

```diff
@@ -1,20 +1,13 @@
-name: Lint
+name: Action lint
 
 on:
   pull_request:
 
 jobs:
-  actionlint:
-    name: actionlint
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: bash
+  call-action-lint-workflow:
+    runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v3
-
-      - name: Check workflow files
-        run: |
-          echo "::add-matcher::.github/actionlint-matcher.json"
-          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/v1.6.19/scripts/download-actionlint.bash)
-          ./actionlint -color
+      - name: Lint actions and workflows
+        uses: bakdata/ci-templates/actions/action-lint@838fc43bf3cdd82b253a21c2f98157b449c7253b
```

Comment:
ToDo: change to release version

Reply:
Needs to be a separate PR because the commit is part of this PR